<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourcePanel</class>
 <widget class="QMainWindow" name="ResourcePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="nrw_reorces_tree_container"/>
    </item>
    <item>
     <widget class="QComboBox" name="cmbConnection"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddAsGeoJSON"/>
   <addaction name="actionAddWFS"/>
   <addaction name="separator"/>
   <addaction name="actionOpenMapInBrowser"/>
   <addaction name="separator"/>
   <addaction name="actionCreateNewGroup"/>
   <addaction name="actionImportQGISProject"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionAddAsGeoJSON">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add GeoJSON</string>
   </property>
   <property name="toolTip">
    <string>Add vector layer as GeoJSON</string>
   </property>
  </action>
  <action name="actionAddWFS">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add WFS</string>
   </property>
   <property name="toolTip">
    <string>Add WFS layers</string>
   </property>
  </action>
  <action name="actionCreateNewGroup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create new group</string>
   </property>
   <property name="toolTip">
    <string>Create new resource group</string>
   </property>
  </action>
  <action name="actionImportQGISProject">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import current project</string>
   </property>
   <property name="toolTip">
    <string>Import current qgis project</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionOpenMapInBrowser">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open map in browser</string>
   </property>
   <property name="toolTip">
    <string>Open map in browser</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
